hs_core/models.py

```diff
@@ -526,6 +526,9 @@ def update(cls, element_id, **kwargs):
             element.metadata.is_dirty = True
             element.metadata.save()
 
+        if(type(kwargs['abstract']) == list):
```

Comment:
Same question why ```kwargs['abstract']``` is a list, and if it is in hs_restclient, it should probably be commented/documented here.